migration/idempotent/001-base.sql

```diff
@@ -316,6 +358,8 @@ BEGIN
    EXECUTE format('GRANT SELECT ON TABLE prom_data_series.%I TO prom_reader', NEW.table_name);
    EXECUTE format('GRANT SELECT, INSERT ON TABLE prom_data_series.%I TO prom_writer', NEW.table_name);
    EXECUTE format('GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE prom_data_series.%I TO prom_modifier', NEW.table_name);
+   EXECUTE format('ALTER TABLE %I.%I OWNER TO prom_modifier', NEW.table_schema, NEW.table_name);
+   EXECUTE format('ALTER TABLE prom_data_series.%1$I OWNER TO prom_modifier', NEW.table_name);
```

Comment:
FYI these two lines are the ones which cause a problem in multi-node and which John is currently working on addressing.

Reply:
now wondering whether `prom_modifier` should be the owner or `prom_admin`

my `set role prom_modifier; create table...` won't work. can't set role in a security definer context.

Reply:
I've opened https://github.com/timescale/promscale_extension/issues/138 to track.